openxr/src/entry.rs

```diff
@@ -108,7 +108,31 @@ impl Entry {
         Ok(f.unwrap())
     }
 
-    /// Create an OpenXR instance with certain extensions enabled
+    /// Initialize Android loader. This must be called before `create_instance()`.
+    #[cfg(target_os = "android")]
+    pub fn initialize_android_loader(&self) -> Result<()> {
```

Comment:
Could we do this automatically at the start of `create_instance`?

Reply:
It would make more sense to insert this call inside `Entry::load()`, the problem is that they return different error types. While not totally correct, it is possible to insert this inside `create_instance`, and I also verified that calling this twice does not have any adverse impact.

Reply:
Hmm, I'd rather not rely on coincidence there; perhaps keeping it separate is the most practical option.